include/fluent-bit/flb_byteswap.h

```diff
@@ -102,4 +103,13 @@ static inline uint64_t FLB_BSWAP_64(uint64_t value)
 
 #endif
 
+static inline uint32_t FLB_TO_NATIVE_UINT32(uint32_t value)
```

Comment:
Please use `FLB_BYTE_ORDER` and `FLB_BIG_ENDIAN` instead of these.

Reply:
Please rename this to `FLB_UINT32_TO_HOST_BYTE_ORDER` so its intention is clear.

Reply:
Done.